docs/sdkx-react-native/getting-started.mdx

```diff
@@ -0,0 +1,179 @@
+
+<Image src="/static/books/react-native/attachment_picker_ios.png" width="half" />{" "}
+
+To use the photo library, Apple requires the app to have Usage Description strings in the Info.plist while. Failing to add these strings might result in app rejection. The following strings are needed for the attachment feature:
+
+<table>
+  <tr>
+    <th>Key</th>
+    <th>Suggested string</th>
+    <th>Notes</th>
+  </tr>
+  <tr>
+    <td>NSPhotoLibraryUsageDescription</td>
+    <td>
+      “We need to access photos library to allow users manually pick images
+      meant to be sent as an attachment for help and support reasons.”
+    </td>
+    <td>
+      This is not needed if your app is iOS 11 or above. Below iOS 11, this key
+      is compulsory else the app may crash when user tries to open the photo
+      library for attaching photos.
+    </td>
+  </tr>
+</table>
+
+Note that the above strings are just a suggested description. If you need localisations for the same, please [Contact Us](https://support.helpshift.com/hc/en/13-helpshift-technical-support/contact-us/?p=all&contact=1)
+
+<Admonition type="info" title="Note">
+
+End-users can attach files such as pdf, video, etc in their issues. For iOS 10 and below, to access files in the “Files” app of iOS, developers will need to add iCloud capability with iCloud Documents services enabled. For more info please refer the [Prerequisites section here.](https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/DocumentPickerProgrammingGuide/Introduction/Introduction.html#//apple_ref/doc/uid/TP40014451)
+&nbsp;&nbsp; <Image src="/static/books/ios/icloud-entitlement.png"  width="full" />
+
+</Admonition>
+
+## Initializing Helpshift in your app {#initializing}
+
+1. First, create an app on the Helpshift Dashboard
+
+   <Image src="/static/books/helpshiftx/add_app.png" width="full" />{" "}
+
+2. Create an app with **Android** and **iOS** as selected `Platform`
+   <Image
+     src="/static/books/helpshiftx/create_app_with_android_platform.png"
+     width="full"
+   />{" "}
+
+Helpshift uniquely identifies each registered App with a combination of 2 tokens:
+
+<div className=" compact">
+
+|                          |                                                                                            |
+| ------------------------ | ------------------------------------------------------------------------------------------ |
+| <code>Domain Name</code> | Your Helpshift domain. E.g. _happyapps.helpshift.com_                                      |
+| <code>Platform ID</code> | Your App's unique Platform ID (App's <code>App ID</code> on dashboard is your Platform ID) |
+| <code>Config</code>      | Config <code>object</code> representing the configuration parameters for the installation. |
```

Comment:
I dont think we need to mention config here since here we are talking about 2 tokens to uniquely identify an app.